include/libk++/align.h

```diff
@@ -2,33 +2,17 @@
 
 #include <stdint.h>
 
-namespace firefly::libkern
-{
-    /*
-        align<size_t>(var); - Align `var` on a 4096 byte boundary
-    */
-    template <typename T>
-    inline void align4k(T& in)
-    {
-        in = (in + 4096 - 1) & ~(4096 - 1);
-    }
+#define ALIGN_UP(__number) (((__number) + 4096 - 1) & ~(4096 - 1))
```

Comment:
These could be constexpr functions, avoid macros where possible.

Reply:
Huh, seems like we didn't even need them.
Must've missed them while removing temp code to get everything up and running